app/uk/gov/hmrc/cdsreimbursementclaim/models/package.scala

```diff
@@ -19,13 +19,16 @@ package uk.gov.hmrc.cdsreimbursementclaim
 import cats.data.Validated.Invalid
 import cats.data.{NonEmptyList, ValidatedNel}
 import play.api.libs.json.{Format, JsonValidationError, Reads, Writes}
+import uk.gov.hmrc.cdsreimbursementclaim.models.claim.Claim
 
 package object models {
 
   type Validation[A] = ValidatedNel[String, A]
 
   def invalid[A](error: String): Validation[A] = Invalid(NonEmptyList.one(error))
 
+  type ClaimsAnswer = NonEmptyList[Claim]
```

Comment:
@gabor-dorcsinecz please can you move it to `answers` package

Reply:
Done